### PR TITLE
List VPCs in an account

### PR DESCRIPTION
**Pull request by fishnix**

### CODE REVIEW

File: api/handlers_vpc.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/YaleSpinup/apierror"
+	"github.com/YaleSpinup/ec2-api/ec2"
+	"github.com/gorilla/mux"
+)
+
+func (s *server) VpvListHandler(w http.ResponseWriter, r *http.Request) {
```
Comment (btassone) [inline api/handlers_vpc.go:13]: :eyes: Vpc? no?

File: api/routes.go [hunk 2]
```diff
@@ -53,7 +53,7 @@ func (s *server) routes() {
 	api.HandleFunc("/{account}/images", s.ImageListHandler).Methods(http.MethodGet).Queries("name", "{name}")
 	api.HandleFunc("/{account}/images", s.ImageListHandler).Methods(http.MethodGet)
 	api.HandleFunc("/{account}/images/{id}", s.ImageGetHandler).Methods(http.MethodGet)
-	api.HandleFunc("/{account}/vpcs", s.ProxyRequestHandler).Methods(http.MethodGet)
+	api.HandleFunc("/{account}/vpcs", s.VpvListHandler).Methods(http.MethodGet)
```
Comment (btassone) [inline api/routes.go:56]: :eyes: Vpc? No?

**Review by btassone — APPROVED** [review]
Works locally. Just the 2 spots that should be renamed.

File: api/handlers_vpc.go (hunk 1 above)
Comment (fishnix) [reply api/handlers_vpc.go:13]: good catch - damn IDE auto complete

**Pull request merged**